cpp/oneapi/dal/algo/basic_statistics/test/batch.cpp

```diff
@@ -33,24 +33,29 @@ TEMPLATE_LIST_TEST_M(basic_statistics_batch_test,
     SKIP_IF(this->not_float64_friendly());
 
     const te::dataframe data =
-        GENERATE_DATAFRAME(te::dataframe_builder{ 100, 10 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 200, 20 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 200, 530 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 500, 250 }.fill_normal(0, 1, 7777),
-                           te::dataframe_builder{ 6000, 20 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 6000, 530 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 10000, 200 }.fill_normal(-30, 30, 7777),
-                           te::dataframe_builder{ 1000000, 20 }.fill_normal(-0.5, 0.5, 7777));
-
-    bs::result_option_id res_min_max = result_options::min | result_options::max;
-    bs::result_option_id res_mean_varc = result_options::mean | result_options::variance;
-    bs::result_option_id res_all = bs::result_option_id(dal::result_option_id_base(mask_full));
+        GENERATE_DATAFRAME(te::dataframe_builder{ 2, 10 }.fill_normal(0, 1, 777),
+                           te::dataframe_builder{ 10, 10 }.fill_normal(-1, 1, 777),
+                           te::dataframe_builder{ 100, 10 }.fill_normal(-30, 30, 777),
+                           te::dataframe_builder{ 200, 530 }.fill_normal(-30, 30, 777),
+                           te::dataframe_builder{ 6000, 530 }.fill_normal(-30, 30, 777));
```

Comment:
What is the reason for such dramatic data size reduction?

Reply:
All dimensions have been restored and new added